x-pack/plugins/cases/server/client/cases/find.test.ts

```diff
@@ -63,4 +63,23 @@ describe('find', () => {
       expect(call.caseOptions).not.toHaveProperty('rootSearchFields');
     });
   });
+
+  describe('throwErrorIfSearchFieldsAreInvalid', () => {
+    const clientArgs = createCasesClientMockArgs();
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('invalid searchFields', async () => {
+      const searchFields = ['foobar'];
```

Comment:
Just to be sure how about we create another test where we use `foobar` not within an array.